resources/lang/en/entities.php

```diff
@@ -166,6 +166,15 @@
     'chapters_permissions_active' => 'Chapter Permissions Active',
     'chapters_permissions_success' => 'Chapter Permissions Updated',
     'chapters_search_this' => 'Search this chapter',
+    'chapters_sort' => 'Sort Chapter Contents',
+    'chapters_sort_named' => 'Sort Chapter :chapterName',
+    'chapters_sort_name' => 'Sort by Name',
+    'chapters_sort_created' => 'Sort by Created Date',
+    'chapters_sort_updated' => 'Sort by Updated Date',
```

Comment:
Can just use the existing book-based string instead of creating new translation strings with the same content.